build/teamcity-publish-artifacts.sh

```diff
@@ -9,7 +9,7 @@ export BUILDER_HIDE_GOPATH_SRC=1
 
 build/teamcity-publish-s3-binaries.sh "$@"
 
-if [ "$TC_BUILD_BRANCH" != master ]; then
+if [ "$TC_BUILD_BRANCH" != *alpha* ] && [ "$TEAMCITY_BUILDCONF_NAME" == 'Publish Releases' ]; then
```

Comment:
I believe the first comparison needs double braces to work properly, i.e. `[[ "$TC_BUILD_BRANCH" != *alpha* ]]`